src/views/components-demo/markdown.vue

```diff
@@ -138,6 +138,16 @@ export default class extends Vue {
   private content3 = simpleContent
   private content4 = simpleContent
   private html = ''
+  // Mapping for local lang to tuiEditor lang
```

Comment:
this is moved to component/markdown instead of the demo page here, please remove this and edit it there.